font/src/ft/mod.rs

```diff
@@ -484,6 +522,20 @@ impl FreeTypeRasterizer {
                 }
                 Ok((bitmap.rows(), bitmap.width(), packed))
             },
+            PixelMode::Bgra => {
+                let buf_size = (bitmap.rows() * bitmap.width() * 4) as usize;
+                let mut i = 0;
+                while i < buf_size {
+                    // Convert BGRA to RGB
+                    //
+                    // XXX our rendring works in rgb and doens't care about urers alpha.
+                    packed.push(buf[i + 2]);
```

Comment:
I guess we should perform this conversion right here, the question is whether we want to keep alpha data, it's useless for alacritty, but we can drop it there and not here.

Reply:
I mean drop it in renderer.rs.